subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/adb/Device.kt

```diff
@@ -10,7 +10,14 @@ class Device(
     private val serial: String,
     private val logger: (String) -> Unit = {}
 ) {
+    private val androidHome: String? = System.getenv("ANDROID_HOME")
+    private val adb: String = "$androidHome/platform-tools/adb"
```

Comment:
Take a look at `:gradle:android` module, it is a place for this constants
https://github.com/avito-tech/avito-android/blob/develop/subprojects/gradle/android/src/main/kotlin/com/avito/android/AndroidSdk.kt#L18